Gemfile.lock

```diff
@@ -365,7 +365,7 @@ DEPENDENCIES
   database_cleaner (= 1.5.1)
   factory_girl
   foreman (= 0.74.0)
-  gds-api-adapters (= 32.1.0)
+  gds-api-adapters (= 36.0.1)
```

Comment:
This will presumably need to be updated once https://github.com/alphagov/gds-api-adapters/pull/581 and https://github.com/alphagov/gds-api-adapters/pull/587 are merged and released in a newer version of the gem.